src/stirling/source_connectors/tcp_stats/tcp_stats_connector_test.cc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "src/stirling/source_connectors/tcp_stats/tcp_stats_connector.h"
+
+#include "src/common/testing/testing.h"
+
+namespace px {
+namespace stirling {
+
+// Tests TCPStatsConnector::Create() works as expected.
+TEST(TCPStatsConnectorTest, Create) {
+  auto connector = TCPStatsConnector::Create("test_tcp_stats_connector");
+  ASSERT_TRUE(connector != nullptr);
+  EXPECT_OK(connector->Init());
+}
```

Comment:
We should write a test that does a little bit more than bring up the source connector.

Perhaps we could try to use `mux_trace_bpf_test.cc:142` i.e. `TEST_F(MuxTraceTest, Capture)` as a starting point.

The basic ingredients are a test fixture that creates the source connector and has a start and stop data transfer thread. Perhaps you can drive the test using some very low level io, i.e. replacing `ASSERT_OK(RunThriftMuxClient());` with some `netcat` usage or similar.

Reply:
Added  unit test case as suggested.